isis/src/qisis/objs/CubeDnView/CubeDnView.cpp

```diff
@@ -115,22 +115,12 @@ namespace Isis {
     connect(m_workspace, SIGNAL( cubeViewportAdded(MdiCubeViewport *) ),
             this, SLOT( onCubeViewportAdded(MdiCubeViewport *) ) );
 
-
-    // !!!!!!!   TODO  LOOK AT THIS       !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Comment:
What happened to this code? Was it moved to a higher level?

Reply:
Directory now emits a signal to CubeDnView saying that an active control network was set. There were duplicate connections in Directory and CubeDnView doing the same thing, so there was no need for this code.